<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExQComboBox</class>
 <widget class="QWidget" name="ExQComboBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExQComboBox</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>731</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>简单地ComboBox(下拉列表框)：</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QCheckBox" name="cbxComboIsEdit">
         <property name="text">
          <string>下拉框为可编辑</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QComboBox" name="comboBoxLeft"/>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btnComboInit">
         <property name="text">
          <string>初始化列表</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnComboClear">
         <property name="text">
          <string>清除列表</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>复杂地ComboBox(项可存储自定义的数据内容)：</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnInitNameAge">
         <property name="text">
          <string>初始化姓名+年龄</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="comboBoxRight"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>QPlainTextEdit(多行可编辑的富文本的编辑器)：</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnPlainTxtAdd">
         <property name="text">
          <string>文本框内容添加到ComboBox</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnPlainTxtClear">
         <property name="text">
          <string>清除文本内容</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="cbxPlainTxtOnlyRead">
         <property name="text">
          <string>富文本的编辑器只可读</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
